pkg/render/compliance.go

```diff
@@ -136,6 +136,13 @@ func (c *complianceComponent) Objects() []runtime.Object {
 			c.complianceServerService(),
 			c.complianceServerDeployment(),
 		)
+	} else {
+		// We still need a serviceaccount in the managed cluster for RBAC requests.
+		complianceObjs = append(complianceObjs,
+			c.complianceServerServiceAccount(),
+			c.complianceServerManagedClusterRole(),
```

Comment:
Are you sure that this is only needed for a managed cluster? Don't we have subject access reviews in the management / standalone clusters as well?

Reply:
complianceServerClusterRole() Also contains these same permissions, as well as a few others that a managed cluster does not need.

Reply:
Gotcha